src/main/java/com/manardenza/console/VisualUserMenu.java

```diff
@@ -0,0 +1,67 @@
+package com.manardenza.console;
+
+import com.manardenza.Main;
+import com.manardenza.entity.AbstractObject;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+
+public class VisualUserMenu {
+
+    private org.slf4j.Logger log = LoggerFactory.getLogger(VisualUserMenu.class);
+
+    public static void outputSplitLine() {
+        System.out.println("**************************************************************");
+    }
+
+    public static <E> void printListInConsole(List<String> header, List<E> options) {
+        if (header != null) {
+            header.forEach(System.out::println);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)